lib/rules/no-irregular-whitespace.js

```diff
@@ -18,13 +18,20 @@ module.exports = function(context) {
     // Module store of errors that we have found
     var errors = [];
 
+    // Comment nodes.  We accumulate these as we go, so we can be sure to trigger them after the whole `Program` entity is parsed, even for top-of-file comments.
+    var commentNodes = [];
+
+    // Lookup the `skipComments` option, which defaults to `true`.
+    var options = context.options[0] || {};
+    var skipComments = typeof options.skipComments === "undefined" ? true : !!options.skipComments;
```

Comment:
I get this, but as the original intent was to preserve ES5 parsers (that don't care about irregular whitespace in comments), and as the comment at the beginning of the original code file stated "in comments too", I figured this made sense.

If that's truly a problem in your book, I can revert that. Then I'll ask StandardJS to have it on by default :wink: